split_to_CMYK.jsx

```diff
@@ -2,104 +2,108 @@
-    addHueSatDes3.putObject(s2t("using"), s2t("adjustmentLayer"), addHueSatDes1);
-    executeAction(s2t("make"), addHueSatDes3, DialogModes.NO);
-//sets hue, sat, lightness
-    var setHueSatDes1 = new ActionDescriptor();
-    var setHueSatDes2 = new ActionDescriptor();
-    var setHueSatDes3 = new ActionDescriptor();
-    var setHueSatList = new ActionList();
-    var setHueSatRef = new ActionReference();
+      addHueSatRef.putClass(s2t("adjustmentLayer"));
+      addHueSatDes3.putReference(c2t("null"), addHueSatRef);
+      addHueSatDes1.putString(s2t("name"), (props.name) + """ color adjust""");
+      addHueSatDes1.putBoolean(s2t("group"), true );
+      addHueSatDes2.putEnumerated(s2t("presetKind"), s2t("presetKindType"), s2t("presetKindDefault"));
+      addHueSatDes2.putBoolean(s2t("colorize"), true);
+      addHueSatDes1.putObject(s2t("type"), s2t("hueSaturation"), addHueSatDes2);
+      addHueSatDes3.putObject(s2t("using"), s2t("adjustmentLayer"), addHueSatDes1);
+      executeAction(s2t("make"), addHueSatDes3, DialogModes.NO);
+      //sets hue, sat, lightness
+      var setHueSatDes1 = new ActionDescriptor();
+      var setHueSatDes2 = new ActionDescriptor();
+      var setHueSatDes3 = new ActionDescriptor();
+      var setHueSatList = new ActionList();
+      var setHueSatRef = new ActionReference();
 
-    setHueSatRef.putEnumerated(s2t("adjustmentLayer"), s2t("ordinal"), s2t("targetEnum"));
-    setHueSatDes1.putReference(c2t("null"), setHueSatRef);
-    setHueSatDes2.putEnumerated(s2t("presetKind"), s2t("presetKindType"), s2t("presetKindCustom"));
-    setHueSatDes3.putEnumerated(s2t("channel"), s2t("channel"), s2t("composite"));
-    setHueSatDes3.putInteger(s2t("hue"), hue);
-    setHueSatDes3.putInteger(c2t("Strt"), sat);
-    setHueSatDes3.putInteger(s2t("lightness"), light);
-    setHueSatList.putObject(s2t("hueSatAdjustmentV2"), setHueSatDes3);
-    setHueSatDes2.putList(s2t("adjustment"), setHueSatList);
-    setHueSatDes1.putObject(s2t("to"), s2t("hueSaturation"), setHueSatDes2);
-    executeAction(s2t("set"), setHueSatDes1, DialogModes.NO);
-    } else {return}
-}
+      setHueSatRef.putEnumerated(s2t("adjustmentLayer"), s2t("ordinal"), s2t("targetEnum"));
+      setHueSatDes1.putReference(c2t("null"), setHueSatRef);
+      setHueSatDes2.putEnumerated(s2t("presetKind"), s2t("presetKindType"), s2t("presetKindCustom"));
+      setHueSatDes3.putEnumerated(s2t("channel"), s2t("channel"), s2t("composite"));
+      setHueSatDes3.putInteger(s2t("hue"), props.h);
+      setHueSatDes3.putInteger(c2t("Strt"), props.s);
+      setHueSatDes3.putInteger(s2t("lightness"), props.v);
+      setHueSatList.putObject(s2t("hueSatAdjustmentV2"), setHueSatDes3);
+      setHueSatDes2.putList(s2t("adjustment"), setHueSatList);
+      setHueSatDes1.putObject(s2t("to"), s2t("hueSaturation"), setHueSatDes2);
+      executeAction(s2t("set"), setHueSatDes1, DialogModes.NO);
+    }; else {return};
+  };
+};
```

Comment:
You can remove all the `;` on line 88, 89, and 90 here.  Not needed, and technically the first one on line 88 between the closing brace and the else statement is a syntax error